src/badguy/stalactite.cpp

```diff
@@ -36,7 +37,7 @@ Stalactite::Stalactite(const ReaderMapping& lisp) :
 {
   countMe = false;
   set_colgroup_active(COLGROUP_TOUCHABLE);
-  SoundManager::current()->preload("sounds/cracking.wav");
+  SoundManager::current()->preload(get_cracking_sound_path());
   SoundManager::current()->preload("sounds/sizzle.ogg");
   SoundManager::current()->preload("sounds/icecrash.ogg");
```

Comment:
The two other sounds should also get an option so they can be changed for Hanging Ball.